plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointColor/index.tsx

```diff
@@ -86,6 +98,22 @@ const PointColor: React.FC<BaseFieldProps<"pointColor">> = ({ value, editMode, o
     });
   };
 
+  useEffect(() => {
+    if (!isActive || !dataID) return;
+    const timer = setTimeout(() => {
+      postMsg({
+        action: "updateDatasetInScene",
+        payload: {
+          dataID,
+          update: { marker: { style: "image", pointColors } },
```

Comment:
Since this component is changing the color of points, the style needs to be point